backend/src/index.ts

```diff
@@ -0,0 +1,168 @@
+import { ApolloServer } from '@apollo/server'; // preserve-line
+import { startStandaloneServer } from '@apollo/server/standalone'; // preserve-line
+import dotenv from 'dotenv';
+// Prisma
+import { PrismaClient } from '@prisma/client'
+const prisma = new PrismaClient() 
+
+// Password security
+import bcrypt from 'bcryptjs';
+import passport from 'passport';
+import { Strategy as LocalStrategy } from 'passport-local';
+import JwtStrategyConfiguration from '../utils/passportSetup.js';
+import jwt from 'jsonwebtoken';
+
+dotenv.config();
+
+// JWT SETUP
+import { Strategy as JwtStrategy, ExtractJwt } from 'passport-jwt';
+
+const opts: any = {};
+opts.jwtFromRequest = ExtractJwt.fromAuthHeaderAsBearerToken();
+opts.secretOrKey = process.env.SECRET;
+passport.use(JwtStrategyConfiguration);
+
+// A schema is a collection of type definitions (hence "typeDefs")
+// that together define the "shape" of queries that are executed against
+// your data.
+const typeDefs = `#graphql
```

Comment:
Move all typeDefs to a seperate file from index.ts. This file will otherwise become quite bloated.